azureadgraph_connector.py

```diff
@@ -15,32 +15,36 @@
 #
 #
 # Phantom App imports
-import phantom.app as phantom
-from phantom.base_connector import BaseConnector
-from phantom.action_result import ActionResult
-# from phantom.vault import Vault
+import grp
+import json
+import os
+import pwd
+import re
+# import random
+# import base64
+import sys
+import time
 
+import phantom.app as phantom
+import requests
+from bs4 import BeautifulSoup, UnicodeDammit
 # from datetime import datetime, timedelta
 from django.http import HttpResponse
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
+
 from azureadgraph_consts import *
-from bs4 import BeautifulSoup, UnicodeDammit
 
-import requests
-# import random
-# import base64
-import sys
-import json
-import time
-import pwd
-import grp
-import os
-import re
+# from phantom.vault import Vault
+
+
 try:
     import urllib.parse as urlparse
 except ImportError:
-    import urlparse
     import urllib
 
+    import urlparse
```

Comment:
I see a few places throughout the code dealing with handling urllib based on whether it's py2 vs. py3. since we no longer need to support py2 this import and all those branches can be simplified. although it's tangential to this PR so it would be fine to track it as a separate future change